src/Core/AdminConsole/Services/IOrganizationService.cs

```diff
@@ -54,7 +54,6 @@ Task<List<Tuple<OrganizationUser, string>>> RevokeUsersAsync(Guid organizationId
     /// <remarks>
     /// This method must target a disabled Organization that has null keys and status as 'Pending'.
     /// </remarks>
-    Task InitPendingOrganization(User user, Guid organizationId, Guid organizationUserId, string publicKey, string privateKey, string collectionName, string emailToken);
```

Comment:
you can probably remove the XML doc above this method as well and include it in the new command's interface method signature.